R/execution.r

```diff
@@ -223,8 +229,15 @@ handle_graphics = function(plotobj) {
         send_plot(last_recorded_plot)
     }
     # need to be set here to capture the size and have it available when the plot is sent
-    attr(plotobj, '.irkernel_width')  <- getOption('repr.plot.width',  repr_option_defaults$repr.plot.width)
-    attr(plotobj, '.irkernel_height') <- getOption('repr.plot.height', repr_option_defaults$repr.plot.height)
```

Comment:
Why are you changing these lines at all?

You just need to add a `attr(plotobj, '.irkernel_height') <- ...` line, and the change above (`mime2repr[[mime]](..., res = attr(plotobj, '.irkernel_res'))`) will make things look correct, right?